torch/csrc/jit/ir.h

```diff
@@ -510,6 +534,27 @@ friend struct Node;
     return output_;
   }
 
+  Graph* pushScope(const Symbol& scope) {
+    scope_stack_.push_back(scope);
+    return this;
+  }
+
+  Graph* pushScope(const std::string& scope) {
+    if (scope.empty()) {
```

Comment:
This seems like an odd thing to do. If you accidentally passed an empty string to the decorator, won't you ignore the push but then unconditionally perform the pop?

Reply:
Good catch. That check is mostly vestigial, removing it.